scripts/setfit/cal_score.py

```diff
@@ -0,0 +1,50 @@
+import argparse
```

Comment:
I'm curious what you need this script for? We have a `create_summary_table.py` [script](https://github.com/huggingface/setfit/blob/main/scripts/create_summary_table.py) for aggregating scores across runs, but maybe you needed something else?

Reply:
oh haha, I didn't see that script 😂
ok, I will remove `cal_score.py` since it's duplicated to `create_summary_table.py`, thanks!

Reply:
done!